### ISSUE TITLE
Use pretrained model on 32kHz dataset to fintune on 16kHz dataset

### ISSUE THREAD

**Issue by qbKhanh** [haoheliu/AudioLDM-training-finetuning#16]
Dear @haoheliu,

First, I want to Thank for your great repo!
I have a naive question that Can I use a pretrained model on 32kHz dataset to fintune on 16kHz dataset.
Hope for your reply!

Best regards,
Khanh

**Comment by haoheliu** [comment]
Yes I think you can use your higher sampling rate audio (>16kHz) to finetune our 16kHz model. That sounds good to me.
